.github/workflows/ci.yml

```diff
@@ -42,8 +42,10 @@ jobs:
       - uses: julia-actions/julia-buildpkg@v1
       - uses: julia-actions/julia-runtest@v1
       - uses: julia-actions/julia-processcoverage@v1
-      - uses: codecov/codecov-action@v1
+      - uses: codecov/codecov-action@v4
         with:
+          fail_ci_if_error: true
+          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
Version 4 of the action does not support tokenless uploads anymore, so a token has to be added as a secret to DiffResults.jl or an organization-wide token to JuliaDiff.